server/grpcsvr/rpc/bmc.go

```diff
@@ -50,8 +51,11 @@ func (b *BmcService) Reset(ctx context.Context, in *v1.ResetRequest) (*v1.ResetR
 			if err != nil {
 				return "", err
 			}
-			// TODO(jacobweinstock) Add task cancellation in taskrunner to honor timeout.
-			taskCtx, _ := context.WithTimeout(context.Background(), defaultTimeout) // nolint
+			taskCtx, cancel := context.WithTimeout(ctx, defaultTimeout)
+			// cant defer this cancel because it cancels the context before the func is run
+			// cant have cancel be _ because go vet complains.
+			// TODO(jacobweinstock): maybe move this context withTimeout into the TaskRunner.Execute function
+			l.V(2).Info(fmt.Sprintf("%v", cancel))
```

Comment:
I think you can get around this by doing `_ = cancel` rather than logging a function reference.

Reply:
oh nice, thanks! will update.